counterparty-core/counterpartycore/server.py

```diff
@@ -227,6 +227,10 @@
             "help": "how long to keep a lock on a UTXO being tracked",
         },
     ],
+    [
+        ("--legacy-api",),
```

Comment:
hm maybe we should do `--enable-v1-api`?

Reply:
ok but it's one or the other of the two versions. Not both at the same time, otherwise you will have to provide another port..

Reply:
ah we can't have the other one available at `/v1/whatever`?

Reply:
we could ofcbut the code is really much cleaner like that! the two APIs are well isolated and we just have to delete the `v1/` folder when we are ready...

Reply:
ah yeah I think we're going to have to support both simultaneously on the same port (with the `/v1/` prefix)

Reply:
The current API code is spaghetti code. I don't know how to do this properly.

Reply:
ach yeah. maybe we should just use a second port for the v1 API

Reply:
yeah sounds good! anyway we must add flags other than `--rpc-*` like `--api-port` etc..